src/app/address/controllers/address.test.js

```diff
@@ -58,45 +58,6 @@ describe("address controller", () => {
         })
       );
     });
-
-    it("should prepopulate with the previous address when in editing previous address route", () => {
-      const generatedAddress = addressFactory(2);
-      req.originalUrl = "/previous/address/edit";
-      req.sessionModel.set("addresses", generatedAddress);
-      address.getValues(req, res, next);
-      expect(next).to.have.been.calledOnce;
-      expect(next).to.have.been.calledWith(
-        null,
-        sinon.match({
-          addressPostcode: generatedAddress[1].postalCode,
-          addressHouseNumber: generatedAddress[1].buildingNumber,
-          addressHouseName: generatedAddress[1].buildingName,
-          addressStreetName: generatedAddress[1].streetName,
-          addressLocality: generatedAddress[1].addressLocality,
-          addressYearFrom: Number(generatedAddress[1].validFrom),
-          addressFlatNumber: generatedAddress[1].buildingNumber,
-        })
-      );
-    });
-
-    it("should prepopulate with the current address when in editing current address route", () => {
```

Comment:
Loads of tests are now no longer needed. As we dont need to tests specific logic when /previous/edit is hit.